packages/core/src/VRange.ts

```diff
@@ -1,6 +1,6 @@
 import { VNode } from './VNodes/VNode';
 import { RelativePosition, Direction, withMarkers } from '../../utils/src/range';
-import { RangeNode } from './VNodes/RangeNode';
+import { RangeNode } from '../../plugin-range/src/VNodes/RangeNode';
```

Comment:
`VRange` should not be aware of `RangeNode` given that it's optional.

Reply:
Actually shouldn't `RangeNode` stand among `VElement` and `FragmentNode` as a generic node type? That is, not in a plugin. It's not exactly optional, is it?